app/src/main/java/com/android/sample/ui/map/Map.kt

```diff
@@ -131,4 +155,119 @@ fun MapScreen(
             tabList = LIST_TOP_LEVEL_DESTINATION,
             selectedItem = Route.MAP)
       })
+
+  if (showBottomSheet) {
+    ModalBottomSheet(
+        onDismissRequest = { showBottomSheet = false },
+    ) {
+      Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
+        // Ligne pour la croix en haut à droite
```

Comment:
For consistency with the rest of the project, it could be great to put the comments in English :)

Reply:
my bad, I will remove it